src/views/SpaceSettings.vue

```diff
@@ -645,9 +612,19 @@ onMounted(() => {
         </Block>
       </template>
     </template>
-    <template v-if="(loaded && isOwner) || (loaded && isAdmin)" #sidebar-right>
-      <div class="lg:fixed lg:w-[321px]">
+    <template #sidebar-right>
+      <div
+        v-if="(loaded && isSpaceOwner) || (loaded && isSpaceAdmin)"
+        class="lg:fixed lg:w-[300px]"
+      >
         <Block :title="$t('actions')">
+          <router-link
+            :to="{ name: 'setup', params: { ensAddress: spaceKey } }"
+          >
+            <UiButton class="block w-full mb-2">
+              {{ $t('settings.editController') }}
+            </UiButton>
+          </router-link>
```

Comment:
Shouldn't this button only be shown to the domain owner? Currently `isSpaceOwner` is actually the controller, right? Do we maybe need a separate `isSpaceController`?